modules/heat_conduction/test/tests/sideset_heat_transfer/tests

```diff
@@ -0,0 +1,28 @@
+[Tests]
```

Comment:
We need to add requirements, design, issues here. I'm not sure why this passed the check but rest assured, we are on to you. Tag @aeslaughter.